### PR TITLE
[Localization] Change a few Evil Admin lines to sound more coherent. 

### PR DESCRIPTION
**Pull request by Blitz425**
## What are the changes the user will see?
New lines for select evil admins.

## Why am I making these changes?
Some of the lines used had grammar issues, or straight up did not make sense in the context of Rogue or at all. Aqua Admin Matt was using a pre battle line when he is defeated for some reason as an example. Majority of these lines are taken directly from Bulbapedia. 

## What are the changes from a developer perspective?
Archer, Courtney, Matt, Zinzolin, and Rood all got changes to their dialogue. 

## How to test the changes?
Set an override to an Admin spawn and see.
## Checklist
- [x] **I'm using `beta` as my base branch**
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [ ] Have I considered writing automated tests for the issue?
- [ ] If I have text, did I add placeholders for them in locales?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [ ] Have I provided screenshots/videos of the changes?

### CODE REVIEW

File: src/locales/de/dialogue.ts [hunk 1]
```diff
@@ -482,7 +482,7 @@ export const PGMdialogue: DialogueTranslationEntries = {
       3: "Was machst du hier? Bist du uns gefolgt?"
     },
     "victory": {
-      1: "Na gut, bis der Boss Zeit für dich hat, werde ich dein Gegner sein!",
+      1: "Muwuhahaha! That battle was fun even though I lost!",
```
Comment (CodeTappert) [inline src/locales/de/dialogue.ts:485]:
```suggestion
      1: "Bwaarharharharhar!! Ich hab zwar verloren, aber das hat Spaß gemacht!",
```
https://abcboy101.github.io/poke-corpus/#id=oras.common.23.2332

**Comment by returntoice** [comment]
I request some modifications on Korean locales too

```
"courtney": {
    "encounter": {
      1: "…그 …네가 가진 …우리…마그마단이 원하는 것",
      2: "…………그럼 …………삭제하겠습니다",
      3: "……애널라이즈 ……하고 싶어 ……아하하하♪"
    },
    "victory": {
      1: "……하아하아… ……으 …하아하아…",
      2: `………예상대로 ………예상외 ………너 …………타깃 록 ………했으니까
         $……엑스페리먼트 ……할 테니까 ………너를………계속………아핫…♪`,
      3: "……예상외 ………… ……역시 ……너…재미있어…! ………아하하…♪"
```

```
 "matt": {
    "encounter": {
      1: "…그래서 말이지, 리더님이 시간이 나실 때까진 내가 상대해 주마!",
      2: "후우하앗!! 맥시멈!! 끓어어!! 오르기 시작했다아아앗!!!!! 그래! 그래! 그래애앳!!! 힘이 다할 때까지 해보자아아!!!",
      3: "UPAAAAA!!! 짓뭉개주마아아아아!!"
    },
    "victory": {
      1: "흐~하하하핫!!! 졌지만 즐거운 승부였다!",
      2: `확 확 느껴지는데! 네놈들의 강함이 말이야!
          $제대로 붙었다고 하기엔 조금 모자라지만 이제 타임오버 같네…`,
      3: "오우오웃! 흥분되는 패배로다!"
```

```
"zinzolin": {
    "encounter": {
      1: "너는 플라스마단에게 있어 불안요소가 될 것이다. 여기서 제거하겠다!",
      2: "이렇게 말해도 포기를 못 하나? 그렇다면 마지막 선고를 하는 것이 내 나름의 친절이지!",
      3: "여기까지 오다니 대단한 트레이너군. 그러나 여기가 끝이다."
    },
    "victory": {
      1: "게치스님… 제가 실패했습니다…",
      2: "힘들구나. 나는 떨고 있다. 괴롭지만 살아 있다. 그것이야말로 살아 있다는 실감!",
      3: "흐음. 의외로 똑똑한 트레이너군. 하지만 생각만큼은 아니야."
    }
```

```
 "rood": {
    "encounter": {
      1: "너는 플라스마단에 위협이 되는구나. 너라는 트레이너가 어떤 인물인지 승부로 알아봐야겠다.",
      2: "넌 포기할 줄을 모르는 모양이구나. 그러나 누구도 우리의 계획을 방해할 수는 없다.",
      3: "너는 여기까지 온 것으로 보니 뛰어난 트레이너구나. 그렇다만 이젠 끝이다."
    },
    "victory": {
      1: "게치스님… 임무를 실패했습니다…",
      2: "나는 떨고 있다. 우리는 괴롭지만 이겨냈다.",
      3: "음… 너는 재능이 있는 트레이너구나. 하지만 충분하지는 않다."
    }
```

File: src/locales/en/dialogue.ts [hunk 2]
```diff
@@ -448,23 +448,23 @@ export const PGMdialogue: DialogueTranslationEntries = {
...
       2: "Hehehe... So, I lost, too...",
       3: "Ahya! How could this be? For an Admin like me to lose to some random trainer..."
     },
   },
   "courtney": {
     "encounter": {
-      1: "The thing...The thing that you hold...That is what... That's what we of Team Magma seek...",
-      2: "... Well then...Deleting...",
-      3: "...Ha. ...Analyzing... ...Hah♪"
+      1: "Don't. Get. In. My. Way.",
```
Comment (CodeTappert) [inline src/locales/en/dialogue.ts:459]: So can you add this please also to german? thank

Comment (flx-sta) [reply src/locales/en/dialogue.ts]: No more relevant I'd say

**Comment by flx-sta** [comment]
@Blitz425 I've resolved those translation conflicts for you

File: src/locales/en/dialogue-male.json [hunk 3]
```diff
@@ -381,7 +381,7 @@
   },
   "archer": {
     "encounter": {
-      "1": "Before you go any further, let's see how you far against us, Team Rocket!",
+      "1": "Before you go any further, let's see how you fair against us, Team Rocket!",
```
Comment (AsdarDevelops) [inline src/locales/en/dialogue-male.json:384]:
```suggestion
      "1": "Before you go any further, let's see how you fare against us, Team Rocket!",
```

File: src/locales/en/dialogue-female.json [hunk 4]
```diff
@@ -381,7 +381,7 @@
   },
   "archer": {
     "encounter": {
-      "1": "Before you go any further, let's see how you far against us, Team Rocket!",
+      "1": "Before you go any further, let's see how you fair against us, Team Rocket!",
```
Comment (AsdarDevelops) [inline src/locales/en/dialogue-female.json:384]:
```suggestion
      "1": "Before you go any further, let's see how you fare against us, Team Rocket!",
```

**Review by AsdarDevelops — CHANGES_REQUESTED** [review]
Fixed typo

**Review by CodeTappert — APPROVED** [review]

**Review by returntoice — CHANGES_REQUESTED** [review]
though the scripts for Zinzolin and Rood have returned to the original, I think it would be better to modify Korean scripts as I mentioned

**Comment by CodeTappert** [comment]
> though the scripts for Zinzolin and Rood have returned to the original, I think it would be better to modify Korean scripts as I mentioned

Would you be fine doing this in a seperate pr?

**Review by flx-sta — APPROVED** [review]

**Review by CodeTappert — APPROVED** [review]

**Pull request merged**